scripts/generate_schemas.py

```diff
@@ -200,32 +204,63 @@ def __init__(self, version, schemaDir, outputFile):
         self.schemaDir = schemaDir
         self.outputFile = outputFile
         self.classes = []
-        for f in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
-            self.classes.append(SchemaClass(f))
+        for avscFile in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
+            self.classes.append(SchemaClass(avscFile))
+        self.requestClassNames = \
```

Comment:
We should prefer line breaking using brackets and hanging indents here over using the `\`. Here, we'd have
```python
self.requestClassNames = [
    cls.name for cls in self.classes if re.search(
        'Search.*Request', cls.name)]
```
We've used `cls` as the convention for referring to classes elsewhere in the code.